CMakeLists.txt

```diff
@@ -141,6 +141,11 @@ if(PLUGIN_ACTIVITYMONITOR)
 endif()
 
 if(PLUGIN_TELEMETRY)
+    if (BUILD_ENABLE_TELEMETRY_LOGGING)
+        message("Building with telemetry logging")
+        add_definitions (-DENABLE_TELEMETRY_LOGGING)
+    endif()
+
```

Comment:
Please remove changes related to BUILD_ENABLE_TELEMETRY_LOGGING. Flags needed for testing can be added through the test's CMAKE_CXX_FLAGS in yml.